pkg/vault/test/foundry/CompositeLiquidityRouterNestedPools.t.sol

```diff
@@ -1467,11 +1515,9 @@ contract CompositeLiquidityRouterNestedPoolsTest is BaseERC4626BufferTest {
         );
 
         // Check ChildPoolB.
-        // Since DAI amountOut comes from parentPool and childPoolB, we need to calculate the proportion that comes
-        // from childPoolB.
         assertApproxEqAbs(
             vars.childPoolBAfter.dai,
-            vars.childPoolBBefore.dai - (amountsOut[vars.daiIdx] - poolInitAmount.mulDown(proportionToRemove)),
+            vars.childPoolBBefore.dai - amountsOut[vars.daiIdx],
```

Comment:
This was redundant right?

Reply:
We used to have DAI in there twice, so all the DAI calculations were "doubled". Now tokens are only there once, so the calculations are simpler.